verification/curator-service/ui/src/components/EpidNavbar.tsx

```diff
@@ -34,23 +30,16 @@ export default function EpidNavbar() {
         history.push('/');
     };
 
-    const handleAddCase = () => {
-        history.push('/new');
-    };
-
     return (
         <div className={classes.root}>
             <AppBar position="static">
                 <Toolbar>
-                    <IconButton onClick={handleHomeClick} edge="start" className={classes.menuButton} color="inherit" aria-label="menu">
+                    <IconButton data-testid="home-btn" onClick={handleHomeClick} edge="start" className={classes.menuButton} color="inherit" aria-label="menu">
```

Comment:
nit: line is getting pretty long, consider breaking into multiple

Reply:
done.